src/kernel/manager.ts

```diff
@@ -67,8 +79,8 @@ class KernelManager implements Kernel.IManager {
       return;
     }
     this._isDisposed = true;
-    clearTimeout(this._updateTimer);
-    clearTimeout(this._refreshTimer);
+    clearTimeout(this._runningTimer);
+    clearTimeout(this._specsTimer);
```

Comment:
Technically, these calls will work: https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/clearTimeout

But because these are intervals, it's better to use `clearInterval`.